pkg/csi/cinder/nodeserver.go

```diff
@@ -561,3 +588,22 @@ func getNodeID(mount mount.IMount, metadata openstack.IMetadata) (string, error)
 	}
 	return nodeID, nil
 }
+
+func verifyTargetDir(target string) error {
```

Comment:
looks like this function is not specific for dir? why name this as Dir?

Reply:
Removed the function entirely, since it was used in one specific case.